src/main/java/io/stargate/sgv2/jsonapi/config/constants/DocumentConstants.java

```diff
@@ -8,31 +8,40 @@ interface Fields {
     /** Primary key for Documents stored; has special handling for many operations. */
     String DOC_ID = "_id";
 
-    /**
-     * Atomic values are added to the array_contains field to support $eq on both atomic value and
-     * array element
-     */
-    String DATA_CONTAINS = "array_contains";
-
-    /** Text map support _id $ne and _id $nin on both atomic value and array element */
-    String QUERY_TEXT_MAP_COLUMN_NAME = "query_text_values";
-
-    /** Physical table column name that stores the vector field. */
-    String VECTOR_SEARCH_INDEX_COLUMN_NAME = "query_vector_value";
-
     /** Document field name to which vector data is stored. */
     String VECTOR_EMBEDDING_FIELD = "$vector";
 
     /** Document field name that will have text value for which vectorize method in called */
     String VECTOR_EMBEDDING_TEXT_FIELD = "$vectorize";
 
+    /** Document field name that will have lexical (BM-25) content for analyzed text for search */
+    String LEXICAL_CONTENT_FIELD = "$lexical";
+
     /** Document field name that will have text value for which vectorize method in called */
     String BINARY_VECTOR_TEXT_FIELD = "$binary";
 
     /** Field name used in projection clause to get similarity score in response. */
     String VECTOR_FUNCTION_SIMILARITY_FIELD = "$similarity";
   }
 
+  /** Names of columns in Document-containing Tables */
```

Comment:
... and creating separate `Columns` (besides existing `Fields`) to keep grouping.